### ISSUE TITLE
Images should have alt attribute

### ISSUE THREAD

**Issue by nicksuch** [SamMoher/SamMoher.github.io#2]
Your image currently looks like this:

```html
<img src="sam-moher.jpg">
```

Image tags should have an `alt` attribute, to make them more accessible to things like screen readers, for the visually imparied:

```html
<img src="sam-moher.jpg" alt="Sam Moher is a cool dude">
```
(feel free to have your own fun with the content of the alt attribute :)

For more details, check out: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img
